internal/pkg/manifest/svc.go

```diff
@@ -442,3 +442,62 @@ func (s *stringOrFromCFN) UnmarshalYAML(value *yaml.Node) error {
 	}
 	return nil
 }
+
+func (cfg ImageWithPortAndHealthcheck) exposedPorts(workloadName string) (exposedPort []ExposedPort) {
```

Comment:
nit: (same for the other methods)
```suggestion
func (cfg ImageWithPortAndHealthcheck) exposedPorts(workloadName string) (exposedPorts []ExposedPort) {
```
or
```suggestion
func (cfg ImageWithPortAndHealthcheck) exposedPorts(workloadName string) (ports []ExposedPort) {
```